src/components/UpgradePlanForm/UpgradePlanForm.js

```diff
@@ -11,12 +11,12 @@ const fieldNames = {
 };
 
 class UpgradePlanForm extends React.Component {
-  constructor({ dependencies: { dopplerLegacyClient } }) {
+  constructor({ dependencies: { dopplerLegacyClient }, intl }) {
     super();
 
     /** @type { import('../../services/doppler-legacy-client').DopplerLegacyClient } */
     this.dopplerLegacyClient = dopplerLegacyClient;
-    this.intl = useIntl();
+    this.intl = intl;
```

Comment:
why can't use hook?

Reply:
I have the same question, what is the rationale behind fixing the bug removing the hook?

Reply:
We cannot use hooks inside a class component, this was the error. It breaks the code.

Reply:
https://reactjs.org/docs/hooks-faq.html#should-i-use-hooks-classes-or-a-mix-of-both